src/views/AccessControl.tsx

```diff
@@ -214,21 +218,23 @@ export const AccessControl = () => {
     const filterDataTable = ():Rule[] => {
         const t = textToSearch.toLowerCase().trim()
         let f:Rule[] = filter(rules, (f:Rule) =>
-            (f.Name.toLowerCase().includes(t) || t === "")
+            (f.Name.toLowerCase().includes(t) || f.Description.toLowerCase().includes(t) || t === "")
         ) as Rule[]
-        // if (optionAllEnabled === "enabled") {
-        //     f = filter(rules, (f:Rule) => f.)
-        // }
+        if (optionAllEnable !== "all") {
+             f = filter(rules, (f:Rule) => !f.Disabled)
```

Comment:
Codacy has a fix for the issue: [Replace `·······f·=·filter(rules,·(f:Rule)·=>·!f.Disabled)` with `f·=·filter(rules,·(f:·Rule)·=>·!f.Disabled);`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      f = filter(rules, (f: Rule) => !f.Disabled);
```